x/concentrated-liquidity/internal/swapstrategy/swap_strategy_test.go

```diff
@@ -31,60 +39,6 @@ func (suite *StrategyTestSuite) SetupTest() {
 	suite.Setup()
 }
 
-// TODO: split up this test case to be separate for each strategy.
-func (suite *StrategyTestSuite) TestComputeSwapState() {
```

Comment:
Note: this test is removed because there are now tests for each compute swap step method of various strategies